core/src/main/scala/stryker4s/run/ThresholdChecker.scala

```diff
@@ -0,0 +1,23 @@
+package stryker4s.run
+
+import grizzled.slf4j.Logging
+import stryker4s.config.Config
+import stryker4s.model.MutantRunResults
+
+class ThresholdChecker extends Logging {
+
+  def determineExitCode(runResults: MutantRunResults)(implicit config: Config): Int = {
+    config.thresholds match {
+      case None =>
+        debug("Threshold not configured. Won\'t fail the build no matter how low your mutation score is.")
```

Comment:
Doesn't the String still compile without the \? I think now it also prints a \

Reply:
Actually, apparently both work: with and without the `\`. I'll remove it though.